tests/llmcompressor/transformers/sparsification/test_compress_tensor_utils.py

```diff
@@ -156,60 +158,71 @@ def test_quant_model_reload(format, dtype, tmp_path):
     dataset = "open_platypus"
     concatenate_data = False
     num_calibration_samples = 64
-    output_dir = tmp_path / "oneshot_out"
     splits = {"calibration": "train[:10%]"}
+    empty_model = AutoModelForCausalLM.from_pretrained(model_path, torch_dtype=dtype)
 
     # create a quantized model
     oneshot(
         model=model_path,
         dataset=dataset,
-        output_dir=output_dir,
         num_calibration_samples=num_calibration_samples,
         recipe=recipe_str,
         concatenate_data=concatenate_data,
         splits=splits,
         oneshot_device=device,
+        clear_sparse_session=False,
         precision=dtype,
     )
 
-    model = SparseAutoModelForCausalLM.from_pretrained(
-        tmp_path / "oneshot_out", torch_dtype=dtype
-    )
+    # Fetch the oneshot model
+    model = get_session_model()
+    og_state_dict = model.state_dict()
```

Comment:
Best variable name in our entire codebase!

Reply:
Agreed, although you'll want to use `get_state_dict_offloaded_model` to account for the offloaded case